pkg/asset/installconfig/gcp/validation.go

```diff
@@ -248,3 +248,19 @@ func ValidateEnabledServices(ctx context.Context, client API, project string) er
 	}
 	return nil
 }
+
+// ValidateProjectRegion determines whether the region is valid for the project
+func ValidateProjectRegion(ctx context.Context, client API, project string, region string) error {
+	validRegions, err := client.GetProjectRegions(ctx, project)
+	if err != nil {
+		return err
+	}
+
+	for r := range validRegions {
+		if region == validRegions[r] {
+			return nil
+		}
+	}
+
+	return fmt.Errorf("region: %s not found: %s", region, strings.Join(validRegions, ","))
```

Comment:
`strings.Join(validRegions, ","))` what does the output of this look like? I'm imagining a very long list.

Reply:
It definitely could be a very long list. should we newline separate it? I just noticed that we have some VERY long string errors, so I wasn't entirely sure if this made sense or fit our norms.